lib/tensorflow/graph.rb

```diff
@@ -86,18 +87,45 @@ def define_op(opName, name , input, device, attrs)
     node
   end
 
+  #
+  # Creates a constant Tensor that is added to the graph with a specified name.
+  # Official documentation of {tf.constant}[https://www.tensorflow.org/versions/r0.9/api_docs/python/constant_op.html#constant].
+  #
+  def constant(name, data, type)
+    tensor = Tensorflow::Tensor.new(data, type)
+    self.constants = {name => tensor}
```

Comment:
I'm not 100% sure what this line should do. From code, it seems like several calls can result in unexpected state:
```ruby
graph = TensorFlow::Graph.new
graph.constant( "const1",[634, 33], :float64)
graph.constant( "const2",[332, 34], :float64)
graph.constants
# Will return {'const2' => Tensor}, while I'd expect {'const1' => Tensor, 'const2' => Tensor}
```